bciers/apps/registration/app/data/jsonSchema/operationRegistration/operationRepresentative.ts

```diff
@@ -1,18 +1,148 @@
+  type: "object",
+  title: "Personal Information",
+  required: ["first_name", "last_name"],
+  properties: {
+    first_name: {
+      type: "string",
+      title: "First Name",
+    },
+    last_name: {
+      type: "string",
+      title: "Last Name",
+    },
+  },
+};
+
+const section2: RJSFSchema = {
+  type: "object",
+  title: "Work Information",
+  required: ["position_title"],
+  properties: {
+    position_title: {
+      type: "string",
+      title: "Job Title / Position",
+    },
+  },
+};
+
+const section3: RJSFSchema = {
+  type: "object",
+  title: "Contact Information",
+  required: ["email", "phone_number"],
+  properties: {
+    email: {
+      type: "string",
+      title: "Business Email Address",
+      format: "email",
+    },
+    phone_number: {
+      type: "string",
+      title: "Business Telephone Number",
+      format: "phone",
+    },
+  },
+};
+
+const section4: RJSFSchema = {
+  type: "object",
+  title: "Address Information",
+  properties: {
+    street_address: {
+      type: "string",
+      title: "Business Mailing Address",
+    },
+    municipality: {
+      type: "string",
+      title: "Municipality",
+    },
+    province: {
+      type: "string",
+      title: "Province",
+      anyOf: provinceOptions,
+    },
+    postal_code: {
+      type: "string",
+      title: "Postal Code",
+      format: "postal-code",
+    },
+  },
+};
+
+export const tempContactsSchema: RJSFSchema = {
+  type: "object",
+  required: ["section1", "section2", "section3", "section4"],
+  properties: {
+    section1,
+    section2,
+    section3,
+    section4,
+  },
+};
 
 export const operationRepresentativeSchema: RJSFSchema = {
   title: "Operation Representative",
   type: "object",
   properties: {
-    name: {
-      type: "string",
-      title: "Name",
+    operation_representative_preface: {
+      //Not an actual field in the db - this is just to make the form look like the wireframes
+      type: "object",
+      readOnly: true,
+    },
+    operation_representatives: {
+      type: "array",
+      items: {},
+      title: "Operation Representative",
+    },
+    new_operation_representatives: {
+      type: "array",
+      items: {},
+      title: "Operation Representative",
     },
   },
 };
 
 export const operationRepresentativeUiSchema: UiSchema = {
   "ui:FieldTemplate": FieldTemplate,
   "ui:classNames": "form-heading-label",
+  "ui:order": [
+    "operation_representative_preface",
+    "operation_representatives",
+    "new_operation_representatives",
+  ],
+  operation_representative_preface: {
+    "ui:FieldTemplate": TitleOnlyFieldTemplate,
+    "ui:title": operationRepresentativePreface,
+  },
+
+  operation_representatives: {
+    "ui:widget": "MultiSelectWidget",
+    // RJSF has "ui:help" but it doesn't play nice with the MultiSelectWidget that is why we are using a custom name ("ui:helperText")
+    "ui:helperText": operationRepresentativeAdd,
+    "ui:placeholder": "Select an individual",
+  },
```

Comment:
Adding text that appears outside of the input in the widget itself is problematic and something that is meant to be handled in the `FieldTemplate`. This way we also don't need to roll our own `helperText` and can use the built in `ui:help` as intended. As you can see this causes misalignment of the `label` and the  `error`:

<img width="1383" alt="Screenshot 2024-08-29 at 7 04 08 AM" src="https://github.com/user-attachments/assets/f7b866d8-90f7-41e4-92b2-9bb83ec96380">

Since we haven't used `ui:help` yet our `InlineFieldTemplate` doesn't handle it well yet. I played around with it a bit and came up with this:
```
    <div>
      <div
        className={`flex flex-col md:flex-row items-start md:items-center ${classNames}`}
      >
        {isLabel && (
          <div className="w-full lg:w-3/12">
            <label htmlFor={id} className="font-bold">
              {label}
              {required && "*"}
            </label>
          </div>
        )}
        <div className={`relative flex items-center w-full ${cellWidth}`}>
          {children}
        </div>
        {isErrors && (
          <div
            className="w-full md:w-4/12 flex items-center text-red-600 ml-0 md:ml-4"
            role="alert"
          >
            <div className="hidden md:block mr-3">
              <AlertIcon />
            </div>
            <span>{error}</span>
          </div>
        )}
      </div>
      <div
        className={`flex flex-col md:flex-row items-start md:items-center ${classNames}`}
      >
        {isLabel && <div className="w-full lg:w-3/12" />}
        <div className={`relative flex items-center w-full ${cellWidth}`}>
          {description}
          {help}
        </div>
      </div>
    </div>
```

The fix might seem slightly more complicated but it will keep our widgets isolated and handle this in the template. Now the helper text doesn't effect `label`/`error` alignment:

<img width="1383" alt="Screenshot 2024-08-29 at 7 22 41 AM" src="https://github.com/user-attachments/assets/4e72faff-04a7-4c58-b44e-0988cb72a80b">

Looks like the text is aligned slightly different in the original implementation, I didn't touch the message text itself so that might need adjustment.

Reply:
Thanks for this 🙏